pr_agent/servers/github_app.py

```diff
@@ -80,8 +81,12 @@ async def handle_request(body: Dict[str, Any]):
             return {}
         pull_request = body["issue"]["pull_request"]
         api_url = pull_request.get("url")
+        comment_id = body.get("comment", {}).get("id")
+        provider = get_git_provider()(pr_url=api_url)
+        provider.add_eyes_reaction(comment_id)
```

Comment:
@okotek won't that add the reaction to every comment? Even if it's not even directed towards the PRAgent? Maybe this can move to `PRAgent.handle_request`?